azurerm/internal/services/firewall/validate/firewall_policy_network_rule_port.go

```diff
@@ -0,0 +1,53 @@
+package validate
+
+import (
+	"fmt"
+	"regexp"
+	"strconv"
+)
+
+func FirewallPolicyNetworkRulePort(i interface{}, k string) (warnings []string, errors []error) {
```

Comment:
Instead of creating an entire new function could we add this to the validate package as `validate.PortOrPortRangeWithin(1,65535)` and then both properties can share it?

Reply:
Updated.